content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/api-methodologies.md

```diff
@@ -35,19 +35,191 @@ Learn more about these objectives from OWASP:
+</br>
+
+Our pentesters run several tests in this area, including:
+
+- Automated crawling tests, followed by manual verification
+- Additional manual crawling tests for better coverage
+  - Includes application areas protected by authentication
+  - Where applicable, our pentesters run automated scans with the authenticated session
+
+Our pentesters use extreme caution to minimize impact on the targeted system.
+
+Our pentesters may use scanning tools, including:
+
+- Burp Suite Community/Professional Edition
+- Nikto
+- Nessus
+- OWASP Zed Attack Proxy (ZAP)
+
+{{% /expand %}}
+
+{{% expand "Manual API vulnerability testing and exploitation" %}}
+</br>
+
+Cobalt pentesters use manual testing tools to identify and analyze the following aspects of the API asset:
+
+- Functionality
+- Workflows
+- Business logic
+- Vulnerabilities in deployment and implementationo
+
+The assessment identifies known vulnerabilities, including those listed in the:
+
+- [OWASP API Top 10](https://owasp.org/www-project-api-security/)
+- [CVE](https://cve.mitre.org/)
+
+Our pentesters also:
+
+- Run injection attacks that probe the robustness of server-validation routines
+- Look for session management flaws that may allow user impersonation
+- Investigate flaws in access control that expose data or enable users to gain elevated privileges.
+
+Beyond these tests, our pentesters also:
+
+- Test how well the design and implementation protects data against unauthorized access or disclosure
+- Review how the endpoints validate input
+- Review how the API handles access tokens
+- Review how the API responds to error conditions or invalid states
+- Consider how resistant the API is to accidental misuse or unintended mistakes by a user that would lead to security issues
+
+For microservices, our pentesters focus on the interactions between different systems. Specifically, we examine
+
+- Access control management
+- Cross-Origin Resource Sharing (CORS) implementation
+- Vulnerabilities outlined in the [OWASP API Security Project](https://owasp.org/www-project-api-security/)
+
+Our pentesters identify the risk associated with each finding, based on:
+
+- A demonstration of the exploit
+- An evaluation of the impact on the asset, with respect to:
+  - Business functionality
+  - Data
+  - Users
+
+When our pentesters "exploit" a finding, they demonstrate the presence of the vulnerability
+while minimizing potential adverse impact to the application, its data, and underlying systems.  
+
+Our pentesters may use scanning tools, including:
+
+- Burp Suite Community/Professional Edition
+- sqlmap
+- Postman
+- OWASP Zed Attack Proxy (ZAP)
+
+{{% /expand %}}
+
+{{% expand "Ongoing assessments" %}}
+</br>
+
+Our pentesters report their findings, in real time, through the Cobalt platform.
+They also:
+
+- Assess all risks
+- Recommend steps for remediation
+
+You're welcome to communicate with our pentesters for each of their findings.
```

Comment:
```suggestion
Our pentesters report their findings, in real time, through the Cobalt platform.
They also:

- Assess all risks
- Recommend steps for remediation

You're welcome to communicate with our pentesters for each of their findings.
```

This is another potential shortcode.

Reply:
Hmm... did we release where pentesters report their findings through the platform?

Reply:
@mjang-cobalt No, we don't have a section describing findings yet. Once we have it, we'll add a cross-reference. We can at least mention that they can view findings on the **Findings** tab of the pentest page.